lib/bundler/source/git/git_proxy.rb

```diff
@@ -66,7 +66,7 @@ def version
         def checkout
           if path.exist?
             return if has_revision_cached?
-            Bundler.ui.confirm "Updating #{uri}"
+            Bundler.ui.info "Fetching #{uri}"
```

Comment:
The colored output of `confirm` looked out of place next to the other "Fetching" outputs. It was also unclear to me what the coloring was trying to convey when the same method doesn't use `confirm` when cloning.

Picture from my machine for reference:
<img width="460" alt="bundler" src="https://cloud.githubusercontent.com/assets/301890/9122968/73f44390-3c5a-11e5-8ce1-eed899772a5b.png">